public/templates/agents/visualizations.js

```diff
@@ -44,21 +44,6 @@ export const visualizations = {
           }
         ]
       },
-      {
-        hide: true,
-        vis: [
-          {
-            title: 'Alerts summary',
-            id: 'Wazuh-App-Agents-General-Alerts-summary',
-            width: 60
-          },
-          {
-            title: 'Groups summary',
-            id: 'Wazuh-App-Agents-General-Groups-summary',
-            width: 40
-          }
-        ]
-      }
     ]
   },
   fim: {
```

Comment:
**nitpick**: This file could be deprecated. We should check and remove it if it is not necessary anymore.